make_tests.sh

```diff
@@ -910,7 +910,7 @@ if [ $enable_timeout != 0 ] ; then
 timeout_args="$GNU_TIMEOUT $test_timeout"
 fi
 
-$UNBUFFER $timeout_args $GNU_TIME -o $stat_subtest -f ' EXECUTION_STATUS="OK"\n RETURN_STATUS=%x\n MEM_TOTAL_AVG=%K\n MEM_RESIDENT_AVG=%t\n MEM_RESIDENT_MAX=%M\n CPU_PERCENT=%P\n CPU_ELAPSED_TIME=%E\n CPU_USER_TIME=%U\n CPU_KERNEL_TIME=%S\n PAGE_FAULTS=%F\n FILE_INPUTS=%I\n SOCKET_MSG_REC=%r\n SOCKET_MSG_SENT=%s' $1 >> $log_subtest 2>&1
+$timeout_args $GNU_TIME -o $stat_subtest -f ' EXECUTION_STATUS="OK"\n RETURN_STATUS=%x\n MEM_TOTAL_AVG=%K\n MEM_RESIDENT_AVG=%t\n MEM_RESIDENT_MAX=%M\n CPU_PERCENT=%P\n CPU_ELAPSED_TIME=%E\n CPU_USER_TIME=%U\n CPU_KERNEL_TIME=%S\n PAGE_FAULTS=%F\n FILE_INPUTS=%I\n SOCKET_MSG_REC=%r\n SOCKET_MSG_SENT=%s' $1 >> $log_subtest 2>&1
```

Comment:
This reverts https://github.com/gpac/testsuite/commit/67327c59f43fd32877f291fe507d20cd6c028554, any reason why ?
The rest is OK

Reply:
I will revert this. Where is UNBUFFER used? I remove it because I could not execute bash -x due to this variable not being set.

Reply:
@aureliendavid is this var set only through buildbot ?

Reply:
hey all, 

`UNBUFFER` is used in some scripts to generate better logs, it's defined in an environment variable in these cases and it defaults to empty if not defined

except that if `bash -u` is used it does generates an error when variables aren't defined, so I added an empty default value in make_tests (6dff71e), should be ok now

Reply:
Great thank you!